app/grandchallenge/components/templates/components/partials/ci_example_value.html

```diff
@@ -0,0 +1,36 @@
+{% load json%}
```

Comment:
Please take care with the work you ask for review, I have had to correct this many times

```suggestion
{% load json %}
```

Reply:
Done